browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -7015,6 +7080,15 @@ index c99bcb07366b88ceb0c4f2a8382750f0e2d99b50..8e069a213abfaa4d9be678184eccd293
      auto error = static_cast<NetworkSessionSoup&>(*m_session).soupNetworkSession().checkTLSErrors(url, certificate, tlsErrors);
      if (!error)
          return true;
+@@ -893,7 +895,7 @@ void NetworkDataTaskSoup::didGetHeaders()
+         const char* headerValue;
+         while (soup_message_headers_iter_next(&headersIter, &headerName, &headerValue))
+             requestHeaders.set(String(headerName), String(headerValue));
+-        m_networkLoadMetrics.requestHeaders = WTFMove(requestHeaders);
```

Comment:
Is it safe to delete old statement? Maybe leave both?

Reply:
Done.